### ISSUE TITLE
Fulfilling part of a multi-merchant order removes other parts

### ISSUE THREAD

**Issue by spencern** [reactioncommerce/reaction#3354]
### Expected behavior
Fulfilling a single merchant's part of a multi-merchant order should only mark that part of the order as completed and other merchant's parts of an order should never be modified.

### Actual behavior
When a part of an order is `packed`, the other parts of that order are deleted from the database. My guess is that this is happening because somewhere there's a $set going on with the items in the order from a client that is unaware of the other parts of the order. I've not verified that hunch, and work needs to be done to assess the exact cause of this bug.
See video
http://recordit.co/xw7xI8dgdP

### Steps to reproduce the behavior
1. Setup a multi-merchant marketplace (primary shop + at least one merchant shop)
2. Enable the merchant shop(s)
3. Setup shipping and Stripe payments in the primary shop
4. Setup stripe payments in the merchant shop
5. Create and publish products from each shop
6. As a guest, add products from each merchant to your cart and checkout
7. Fulfill a single merchant's part of the order, from either the merchant or the primary shop.
8. Observe that the other parts of the order are deleted when items are marked as packed

### Versions
```
Node: 8.9.0
NPM: 5.5.1
Meteor Node: 4.8.4
Meteor NPM: 4.6.1
Reaction CLI: 0.23.2
Reaction: 1.5.11
Reaction branch: master
Docker: 17.06.2-ce
```

**Comment by zenweasel** [comment]
I'm unclear what "other parts of the order are deleted" here that are missing.  After packing and shipping the order from the primary site I am still seeing both items on the order. I still seeing two records for billing/shipping and items. It's very possible I am missing something but I think I need a little more clarity about what should be missing.

I am also seeing the order display twice in the sidebar which is definitely a regression.

**Comment by spencern** [comment]
@zenweasel I'll re-check this, sounds like you're not seeing the behavior I'm seeing in the video/gif

**Comment by zenweasel** [comment]
Closed via #3358
